app/api/server/v1/voip/events.ts

```diff
@@ -0,0 +1,31 @@
+import { Match, check } from 'meteor/check';
+
+import { API } from '../../api';
+import { Voip } from '../../../../../server/sdk';
+import { canAccessRoom } from '../../../../authorization/server';
+import { VoipRoom } from '../../../../models/server/raw';
+
+API.v1.addRoute(
+	'voip/events',
```

Comment:
missing the endpoint definition

Reply:
Didnt understand what you mean. @KevLehman Can you please help with this?